lrmi_vocabs/alignmentType/alignmentType.ttl

```diff
@@ -9,53 +9,53 @@
     dct:created "2017-03-01"^^xsd:date ;
-    dct:modified "2017-12-20"^^xsd:date ;    
+    dct:modified "2017-12-20"^^xsd:date ;
     dct:license <http://creativecommons.org/licenses/by/4.0/> .
 
 alignment:assesses a skos:Concept ;
-    skos:prefLabel "assesses"@en-US ;
-    skos:definition "The learning resource being described may be used to assess the competency being referenced."@en-US ;
-    dct:source "Based on Common Education Data Standards (CEDS): https://ceds.ed.gov/element/000869#Assesses."@en-US ;
+    skos:prefLabel "assesses"@en-US, "testet"@de ;
+    skos:definition "The learning resource being described may be used to assess the competency being referenced."@en-US, "Die beschriebene Lernressource kann verwendet werden, um die adressierte Kompetenz zu testen."@de ;
+    dct:source "Based on Common Education Data Standards (CEDS): https://ceds.ed.gov/element/000869#Assesses."@en-US, "Basierend auf den Common Education Data Standards (CEDS): https://ceds.ed.gov/element/000869#Assesses"@de ;
     skos:inScheme alignment: .
-    
+
 alignment:complexityLevel a skos:Concept ;
-    skos:prefLabel "complexity level"@en-US ;
-    skos:definition "The point in the framework being referenced defines a level or range that measures the difficulty or challenge presented by the learning resource being described."@en-US ;
-    skos:scopeNote "Example frameworks include, but are not limited to, Bloom's Taxonomy, Norman Webb's Depth of Knowledge (DOK), Biggs' SOLO taxonomy, Lexile and Quantile frameworks by MetaMetrics."@en-US ;
-    skos:inScheme alignment: . 
+    skos:prefLabel "complexity level"@en-US, "Komplexitätsgrad"@de ;
+    skos:definition "The point in the framework being referenced defines a level or range that measures the difficulty or challenge presented by the learning resource being described."@en-US, "Das Element in dem Bezugssystem, auf das verwiesen wird, definiert ein Niveau oder einen Bereich, der die Schwierigkeit oder Herausforderung der beschriebenen Lernressource misst."@de ;
+    skos:scopeNote "Example frameworks include, but are not limited to, Bloom's Taxonomy, Norman Webb's Depth of Knowledge (DOK), Biggs' SOLO taxonomy, Lexile and Quantile frameworks by MetaMetrics."@en-US, "Beispiele für Bezugssysteme sind unter anderem Bloom's Taxonomy, Norman Webb's Depth of Knowledge (DOK), Biggs' SOLO taxonomy, Lexile and Quantile Frameworks von MetaMetrics."@de ;
+    skos:inScheme alignment: .
 
 alignment:educationalLevel a skos:Concept ;
-    skos:prefLabel "educational level"@en-US ;
-    skos:definition "The point in the framework being referenced defines a level or stage within an education system for which the resource being described is intended or useful."@en-US ;
-    dct:source "Based on Common Education Data Standards (CEDS): https://ceds.ed.gov/element/000869#EducationalLevel."@en-US ;
-    skos:inScheme alignment: . 
-    
+    skos:prefLabel "educational level"@en-US, "Bildungsniveau"@de ;
```

Comment:
in `alignmentType.jsonld` wird "Bildungsstufe" als Übersetzung von educational level genutzt